src/main/data/languages.json

```diff
@@ -2975,6 +2975,15 @@
       "text"
     ]
   },
+  {
+    "dcncLanguage": "Unknown language",
+    "dcncTag": "UND",
+    "rfc5646Tag": "und",
+    "use": [
+      "audio",
+      "text"
+    ]
+  },
```

Comment:
```
    "comments": [
      "Only to be used in fields or elements where the absence of value does not mean 'Unknown language'."
    ],
```

Reply:
addressed in https://github.com/ISDCF/registries/pull/830/commits/845697fa805ea0d8ae721efa49ad0a2fb7783f4b